DirectProgramming/C++SYCL_FPGA/Tutorials/Features/task_sequences/sample.json

```diff
@@ -0,0 +1,71 @@
+{
+  "guid": "AE05DF3B-27CD-4B2E-9698-1BD204AC42D4",
+  "name": "Task Sequences",
+  "categories": ["Toolkit/oneAPI Direct Programming/C++SYCL FPGA/Tutorials/Features"],
+  "description": "An Intel® FPGA tutorial that demonstrates how to use task sequences to achieve asynchronous parallelism within kernel",
+  "toolchain": ["icpx"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["cmake"],
+  "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../..",
+    "include": [
+      "README.md",
+      "Tutorials/Features/task_sequences",
+      "include"
+    ],
+    "exclude": []
+  },
+  "ciTests": {
+    "linux": [
+      {
+        "id": "fpga_emu_2",
```

Comment:
2?

Reply:
Updated to more usable names e.g. fpga_emu_task_sequence